chrome_extension/background/event.js

```diff
@@ -124,7 +116,7 @@ mooltipassEvent.onSaveSettings = function(callback, tab, settings) {
 	mooltipassEvent.onLoadSettings();
 }
 
-mooltipassEvent.onGetStatus = function(callback, tab) {
+mooltipassEvent.onGetStatus = function(callback, senderTab, tab) {
```

Comment:
I may be confused... but why add an arg if you are not using it in the function?